helpers/stylesheet.js

```diff
@@ -2,9 +2,10 @@
 
 const _ = require('lodash');
 const buildCDNHelper = require('./lib/cdnify');
+const ResourceHints = require('./lib/resourceHints');
```

Comment:
lowerCase please or you can directly import function by
`const { addResourceHint } = require('./lib/resourceHints');  `